persistent/ChangeLog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog for persistent
 
+# 2.17.1.0
+
+* [#1600](https://github.com/yesodweb/persistent/pull/1600)
+    * Add `migrateStructured` to `Database.Persist.Postgresql.Internal`.
+      This allows you to access a structured representation of the proposed migrations
+      for use in your application.
```

Comment:
Oh shoot. All the changes in this PR are actually in `persistent-postgresql` package. So this `CHANGELOG` entry needs to be relocated into `persistent-postgresql/CHANGELOG.md` and the relevant versioning followed.